src/utils/telegram-logger.ts

```
@@ -0,0 +1,97 @@
+      body += `\n<b>ERROR:</b> <code>${this.escapeHtml(context)}</code>`;
+    }
+
+    return `${header}\n${body}`;
+  }
+
+  private static async sendMessage(
+    type: 'error' | 'info' | 'warning',
+    message: string,
+    reply_markup?: object,
+  ) {
+    if (!this.botToken) {
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)